src/languages/TypeScript.ts

```diff
@@ -15,6 +15,7 @@ const NODE_TYPES = Object.freeze([
     'type_alias_declaration',
     'property_signature',
     'module',
+    'program'
```

Comment:
Why did we add this line?

Reply:
i added it for testing sth and forgot to remove it